src/components/sidebar/sidebar.css

```diff
@@ -0,0 +1,21 @@
+.sidebar {
+  background-color: white;
+  border-right: 1px solid #D0D0D7; 
+  height: calc(100vh - 94px);
+  width: 100px;
+  position: absolute;
+  left: 0;
+  bottom: 0;
+  display: flex;
+  flex-flow: wrap column;
+  justify-content: center;
+  align-items: center;
+}
+
+.sidebar .material-icons {
+  margin: 10px;
+}
+
+.sidbar a {
```

Comment:
Perhaps you could change sidebar name to something more meaningful?

Reply:
Any suggestions?

Reply:
No